deployment/keystone/changeset/internal/deploy.go

```diff
@@ -377,10 +381,12 @@ func ConfigureOCR3ContractFromJD(env *deployment.Environment, cfg ConfigureOCR3C
 	if !ok {
 		return nil, fmt.Errorf("failed to get contract set for chain %d", cfg.ChainSel)
 	}
-	contract := contracts.OCR3
-	if contract == nil {
-		return nil, fmt.Errorf("no ocr3 contract found for chain %d", cfg.ChainSel)
+
+	contract, err := getOCR3Contract(contracts.OCR3, cfg.Address)
```

Comment:
Maybe add a log line indicating which address are we looking for (or any contract, if empty)